src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/dashboard_grid.tsx

```diff
@@ -7,153 +7,134 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import 'react-resizable/css/styles.css';
-import 'react-grid-layout/css/styles.css';
-
-import { pick } from 'lodash';
 import classNames from 'classnames';
-import React, { useState, useMemo, useCallback, useEffect } from 'react';
-import { Layout, Responsive as ResponsiveReactGridLayout } from 'react-grid-layout';
+import React, { useCallback, useMemo, useRef } from 'react';
 
-import { useBatchedPublishingSubjects } from '@kbn/presentation-publishing';
 import { useAppFixedViewport } from '@kbn/core-rendering-browser';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
```

Comment:
Should just use view mode from `presentation-publishing`.

embeddable-plugin/public viewMode is targeted for removal.

Reply:
Fixed in 822bb967479c79f83950ebae7022304115f4e498